packages/fxa-settings/src/pages/ResetPasswordConfirmed/index.test.tsx

```diff
@@ -23,20 +23,18 @@ describe('ResetPasswordConfirmed', () => {
       'You’re now ready to use Account Settings'
     );
     const passwordResetContinueButton = screen.queryByText('Continue');
-
+    // Calling `getByText` will fail if these elements aren't in the document,
```

Comment:
This comment isn't applicable here since it's using `queryByText`. I'm fine with the other comments being left but I don't think they _need_ comments either since we do grab `getByText` and then `.toBeInTheDocument()` in other places in the codebase.

Reply:
This is referencing the `getByText` calls above -- I'll move it up!

Reply:
Ah oops sorry, missed that.